FirebaseRemoteConfig/Sources/RCNConfigContent.m

```diff
@@ -72,14 +77,16 @@ - (instancetype)initWithDBManager:(RCNConfigDBManager *)DBManager {
     _activeConfig = [[NSMutableDictionary alloc] init];
     _fetchedConfig = [[NSMutableDictionary alloc] init];
     _defaultConfig = [[NSMutableDictionary alloc] init];
+    _activePersonalization = [[NSDictionary alloc] init];
+    _fetchedPersonalization = [[NSDictionary alloc] init];
     _bundleIdentifier = [[NSBundle mainBundle] bundleIdentifier];
     if (!_bundleIdentifier) {
       FIRLogNotice(kFIRLoggerRemoteConfig, @"I-RCN000038",
                    @"Main bundle identifier is missing. Remote Config might not work properly.");
       _bundleIdentifier = @"";
     }
     _DBManager = DBManager;
-    _configLoadFromDBSemaphore = dispatch_semaphore_create(0);
+    _configLoadFromDBSemaphore = dispatch_semaphore_create(1);
```

Comment:
Could you please add a comment why the particular semaphore value is set. Also, maybe it will be a bit more readable if we call `dispatch_semaphore_signal` closer to the operation that requires it?

Reply:
Moved the database methods closer.